docs/tutorials/get-started/blink-an-led.md

```diff
@@ -0,0 +1,416 @@
+You'll create the new component in the **Create component** panel.
+
+- Name the board whatever you like as long as you are consistent when referring to it later; we'll name it `local` since it is the board we will communicate with directly.
+- For the component **Type**, select `board`.
+- For **Model**, select `pi`.
+- Click **Create component**.
+
+Your board component pane will look like this:
+
+<img src="../../img/blink-an-led/board-config.png" alt="Screenshot of the Viam app showing the board configuration on the Config tab. The board is named 'local' and the attributes are shown as empty braces." width="100%">
+
+Click **Save Config** at the bottom left of the page.
+
+## Control your robot using the Viam app
+
+When you configure your board component, the Viam app generates a control panel for it.
+Click the **Control** tab to view the control panels for all your robot's components (in this case, just the board).
+
+Click the board card to expand it.
+Here, you can click on **Get** to get the current status of your pin.
+The first time you click **Get Pin State**, it should return "Pin: 8 is low."
+
+<img src="../../img/blink-an-led/get-pin-state.png" alt="Screenshot of the Viam app showing the board configuration on the Control tab. The 'Board Local' row is expanded, and under the 'Get' row, the pin is set to '8.' A red box is around the 'Get Pin State' button and the output, which reads, 'Pin: 8 is low.'" width="100%">
+
+You can now use the **Set** menu to set the status of your pin to **high**.
+Once you click **Get Pin State** again, it will look like this:
+
+<img src="../../img/blink-an-led/set-pin-state.png" alt="Screenshot of the Viam app showing the board configuration on the Control tab. The 'Board Local' row is expanded, and under the 'Set' row, the pin is set to '8.' A red box is around the 'Set Pin State' field." width="100%">
+
+When you set your pin to **high** the LED should illuminate.
+You can set the pin back and forth between high and low, and you will see your LED turn on or off depending on whether you have the value set to **low** or **high**.
+
+{{<gif webm_src="/tutorials/img/blink-an-led/image3.webm" mp4_src="/tutorials/img/blink-an-led/image3.mp4" alt="A GIF of the completed project showing a blinking blue LED connected to a Raspberry Pi with jumper cables." max-width="110%">}}
+
+Congratulations!
+You have just successfully used Viam to make an LED blink with a Raspberry Pi!
+You have learned how the GPIO on a Raspberry Pi works, and how to build circuits for LED bulbs.
+
+## Control your robot using the Viam SDKs
+
+Now you're ready to control the LED with code so you don't have to click buttons to turn it on and off.
+
+In this section, you will be introduced to the basics of programming hardware by using either the [Viam Python SDK](https://python.viam.dev/) (software development kit) or the [Viam Go SDK](https://pkg.go.dev/go.viam.com/rdk/robot/client#section-readme) to make your LED blink.
```

Comment:
Instead of "will be introduced", keep it about the user and what they will actively learn (instread of passively being introduced to)